contrib/html_demo/JupyterCode/13_image_similarity_export.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<ul><li>Move MAX_SIZE above to where the other parameters are defined, and rename to be more explanatory. E.g. THUMBNAIL_SIZE</li><li>Don't think resize() needs to be defined as a function here since the code is not used anywhere else.</li><li>[minor] consider using fname = os.path.join() since it is more robust to the exact operating system</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Completed